### ISSUE TITLE
Update Readme instructions

### ISSUE THREAD

**Issue by loredanacirstea** [raiden-network/raiden-contracts#62]
After removing populus, we have outdated instructions in the Development section.

E.g.
```
We use populus for development. At the moment, this library is incompatible with recent libraries that act as dependencies. Therefore, we have to use different environments here.
```

- [X] add a link to the official smart contracts specification docs. - https://github.com/raiden-network/raiden-contracts/pull/300
- [X] add disclaimer - https://github.com/raiden-network/raiden-contracts/pull/311
- [X] maybe add a link to the Raiden Network gitter chat? - https://github.com/raiden-network/raiden-contracts/pull/300
Make sure we have up to date instructions for:
- [X] installation - https://github.com/raiden-network/raiden-contracts/pull/304
- [X] usage - https://github.com/raiden-network/raiden-contracts/pull/304
- [X] compilation - https://github.com/raiden-network/raiden-contracts/pull/304
- [X] deployment - https://github.com/raiden-network/raiden-contracts/pull/307
- [X] testing - https://github.com/raiden-network/raiden-contracts/pull/304

**Comment by LefterisJP** [comment]
Can you add some more info to this issue? What do you expect the Readme instructions to contain?

**Comment by rakanalh** [comment]
Here's what i found having tried to use the deploy tool here:

1. I had to execute 
```
 deploy \
       --rpc-provider http://127.0.0.1:8545 \
       --json raiden_contracts/data/contracts.json \
       --owner 0x4581966dab61ee678f25088c0f402664cf044792 \
       --wait 300 \
       --token-name CustomToken --token-symbol TKN \
       --supply 10000000 --token-decimals 18
```
instead of `python -m deploy`... not sure if it was something wrong with my PYTHONPATH. But using the `deploy` command directly worked.

2. I had to realize that the `CustomToken` contract had to be deployed by me and not directly by the deploy tool... which wasn't mentioned in the README. I stumbled upon this when i tried to deploy but was getting a key error on `CustomToken` not being present in the pre-compiled contracts.

3. The tool was complaining about `bytecode` attribute not existing in the contract interface... i was able to fix it by replacing `bytecode` with `bin` in this PR(https://github.com/raiden-network/raiden-contracts/pull/69)
